tests/libtest/thrift_client_to_db.py

```diff
@@ -169,13 +173,19 @@ def __getattr__(self, attr):
 
 class CCProductHelper(ThriftAPIHelper):
 
-    def __init__(self, host, port, uri, auto_handle_connection=True,
-                 session_token=None):
+    def __init__(self, version, host, port, product, uri,
+                 auto_handle_connection=True, session_token=None):
         # Import only if necessary; some tests may not add this to PYTHONPATH.
         from libcodechecker import session_manager
-        from ProductManagement import codeCheckerProductService
 
-        transport = THttpClient.THttpClient(host, port, uri)
+        if version == '6.0':
```

Comment:
Is this a good idea to hardcode this constant? Is it a good idea to also rely on the minor version here?